test/inductor/test_compiled_autograd.py

```diff
@@ -3041,24 +3041,6 @@ def forward(model, x):
 
         expected_logs = [
             "code: CompiledFunctionBackward (NodeCall 2)",
-            "code: CompiledFunctionBackward0 (NodeCall 2)",
```

Comment:
What happened here? I expect there are new logs?

Reply:
The main difference was that in this test, we would previously save a view of a parameter for backward. One consequence of this PR is that we will effectively end up always recomputing views of params in the backward.

@xmfan told me that the main thing this test is testing is for the existence of the `CompiledFunctionBackward` bits of the compiled autograd graph, so it was easier just to make the test less fragile than to manually make the recompute-view tweaks

Horace actually tried to land an identical change half a year ago, although he had some benchmarks showing that this doesn't meaningfully change our OSS benchmarks (if there is an internal test failure I will debug as necessary)

Reply:
sgtm